draft-ietf-protected-audience-key-value-services.md

```diff
@@ -320,28 +320,30 @@ Each key group is expected to have exactly one tag from the following list:
 
 ### Generating a Request {#request-generate}
 
-This section describes how the client MAY form and serialize request messages in order to fetch values from the Trusted Key Value server. 
+This section describes how the client MAY form and serialize request messages in order to fetch values from the Trusted Key Value server.
 
-This algorithm takes as input an [HPKE] `public key` and its associated `key id`, a `metadata` map for global configuration, where both keys and values are strings, and a list of `signals`, each of which is a map, from a list of strings to a list of strings.
+This algorithm takes as input an [HPKE] `public key` and its associated `key id`, a `metadata` map for global configuration, where both keys and values are strings, and a `compression groups` list, each of which is a map, from key `"partitions"` to value `partitions` as a list of maps.
```

Comment:
I'm having trouble parsing what `compression groups` list looks like. What do you mean by "from key `"partitions"` to value `partitions`? I don't see a definition from either type, and I'm not sure what the quote characters in `"partitions`" indicate.